test/avocado/selenium-machines-basic.py

```diff
@@ -0,0 +1,110 @@
+import os
+from avocado import skipIf
+from testlib_avocado.seleniumlib import clickable, invisible
+from machineslib import MachinesLib
+
+
+class MachinesBasicTestSuite(MachinesLib):
+    """
+    :avocado: enable
+    :avocado: tags=machines
+    """
+
+    def testNoVm(self):
+        self.wait_text("No VM is running or defined on this host")
+
+    def testOverviewInfo(self):
+        name = "staticvm"
+        self.create_vm(name)
+
+        self.wait_in_text('256 MiB', '#vm-{}-memory'.format(name))
+        self.wait_in_text('1', '#vm-{}-vcpus-count'.format(name))
+        self.wait_in_text('custom', '#vm-{}-cputype'.format(name))
+        self.wait_in_text('pc', '#vm-{}-emulatedmachine'.format(name))
+        self.wait_in_text('disk,network', '#vm-{}-bootorder'.format(name))
+
+    def testRunVm(self):
+        name = "staticvm"
+        args = self.create_vm(name, state='shut off')
+
+        self.click(self.wait_id('vm-{}-run'.format(name), cond=clickable))
+        self.wait_in_text('running', '#vm-{}-state'.format(name))
+        self.wait_id('vm-{}-run'.format(name), cond=invisible)
+        self.wait_id('vm-{}-reboot'.format(name))
+        self.wait_id('vm-{}-off'.format(name))
+        self.wait_id('vm-{}-delete'.format(name))
+        self.wait_vm_complete_start(args)
+
+    def testRestartVm(self):
+        name = "staticvm"
+        args = self.create_vm(name, wait=True)
+
+        self.click(self.wait_id('vm-{}-reboot'.format(name), cond=clickable))
+        self.machine.execute(
+            'until sudo cat {} | grep "reboot: Power down"; do sleep 1; done'.format(args.get('logfile')))
+        self.wait_in_text('running', '#vm-{}-state'.format(name))
+
+    @skipIf(os.environ.get('HUB') == '10.111.112.10', "Doesn't work when using run-tests")
```

Comment:
is this some bug, why it does not work, or it needs more investigation here? in that case please file the issue after PR will be merged to not forget to look at it later.

Reply:
I think it needs more investigation. The button is clicked, but it's dropdown menu couldn't show up. I have no idea what's the problem.